packages/docusaurus-theme-classic/src/theme/DocItem/index.tsx

```diff
@@ -21,7 +21,7 @@ const LINK_CLASS_NAME = 'table-of-contents__link';
 const ACTIVE_LINK_CLASS_NAME = 'table-of-contents__link--active';
 const TOP_OFFSET = 100;
 
-function DocTOC({headings}) {
+export function DocTOC({headings}) {
```

Comment:
I think we should create a theme/components/TOC/index.tsx and put this component here. We are going to create a theme component library (+ storybook) someday, so it's worth starting with this one.

Importing a doc-related component in a blog related component does not feel the right thing to do.